props/blaster_BC_buttons.h

```diff
@@ -344,6 +342,10 @@ class BlasterBCButtons : public Blaster {
 
   void Setup() override {
     MKSPEC<BCBlasterMenuSpec>::SoundLibrary::init();
+    if (blaster_mode == MODE_AUTO && !SFX_auto) {
```

Comment:
I think you want to do this in SetPreset().
Setup() is only called once on startup.

Reply:
I tried that initially, but I think the sequence of SetPreset(), SetBlasterMode() and ResetCurrentAlternative() all come into play.
It will still use the alt sound of the BLASTER_DEFAULT_MODE unless It's set first at boot in Setup().
Since the only way to get into MODE_AUTO if no auto sounds exist is by the BLASTER_DEFAULT_MODE being set as AUTO, I figured one time at boot was sufficient.

Reply:
But what happens if you change preset from one that has SFX_auto to one that doesn't have SFX_auto?

Reply:
Well, you're right of course. I only tested with one preset.
Would it be ok to move the new call to ResetCurrentAlternative() in chdir() down a line to *after* 
`Effect::ScanCurrentDirectory();`? Or would that mess other things up?

If yes, then I could just put this part in my ResetCurrentAlternative() override instead of needing to add SetPreset() and do this same routine in 2 places.

I tested like that and it works well. But unsure how it might affect other things using current_alternative.

Reply:
Should be fine.

Reply:
done.